diesel/src/macros/mod.rs

```diff
@@ -45,6 +45,13 @@ macro_rules! column {
         {
         }
 
+        impl<ST, Source, Predicate> SelectableExpression<
+            $crate::query_source::filter::FilteredQuerySource<Source, Predicate>, ST>
+            for $column_name where
+            $column_name: SelectableExpression<Source, ST>
```

Comment:
Couldn't find such an impl, so I tried making it myself:
```rust
impl<T, Source, Predicate> SelectableExpression<FilteredQuerySource<Source, Predicate>> for T where
    T: Expression + SelectableExpression<Source>
{
}
```

but it apparently conflicts with [this](https://github.com/robertmaloney/diesel/blob/c124253f50a94727bd64f5b8cc09c2f4e7171449/diesel/src/expression/mod.rs#L120-L124):
```rust
impl<'a, T: ?Sized, ST, QS> SelectableExpression<QS, ST> for &'a T where
    T: SelectableExpression<QS, ST>,
    &'a T: Expression,
{
}
```
Am I missing something?